### ISSUE TITLE
Write the ESRP Release test pipeline

### ISSUE THREAD

**Issue by dkurepa** [dotnet/arcade#11619]
We want to make sure that our ESRP Release task is working as intended. For this purpose, we will create a pipeline which will create some test packages for us, sign them, and publish to the Maven Central Repository. We will use an old, unused folder in the repository (https://repo1.maven.org/maven2/com/microsoft/aspnet/)

**Comment by dkurepa** [comment]
https://dev.azure.com/dnceng/internal/_git/dotnet-release/pullrequest/27408

**Comment by dkurepa** [comment]
The ESRP Release task didn't work straight away, we had to get help from the PRSS/ESRP team:
- We had to upload the authentication certificate to the AAD app that was used for the ESRP signing
- The ESRP team had to allow our app to publish with the `MainPublisher` they told us to use
- Our app had to be allowed to publish to the `com.microsoft.aspnet.publishing-test` and `com.microsoft.aspnet.publishing-test-messagepack` group IDs. We also allowed it to publish to `com.microsoft.signalr` and `com.microsoft.signalr.messagepack` so we don't have to worry about it in the future`
- The ESRP Release task checks the hash files, and they we're causing errors. After some debugging, I found out that it was because we were using the Out-File powershell command, which adds some invisible characters, and a new line to the output file. Using [System.IO.File]::WriteAllText solved this issue

That's it after these changes, the ESRP task worked and we published our first testing package: https://repo1.maven.org/maven2/com/microsoft/aspnet/publishing-test/
